setup.py

```diff
@@ -59,8 +50,10 @@
       download_url = 'https://github.com/RDFLib/sparqlwrapper/releases',
       platforms = ['any'],
       packages = ['SPARQLWrapper'],
-      requires = _requires,
```

Comment:
The whole point is to not depend on pip to get dependencies.
The `requires` field is just useless, `install_requires` is enough.